src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/ThemeServiceTests.cs

```diff
@@ -2879,6 +3189,158 @@ await using (var context = InMemoryContentDbContext(contextId))
             }
         }
 
+        [Fact]
+        public async Task GetPublicationTree_DataCatalogue_ReleaseHasData()
```

Comment:
Seems like this is already covered by: `GetPublicationTree_DataCatalogue_LatestData`. We might want to adjust the name to make it more readable/obvious.

These new assertions should probably be in there instead.